### ISSUE TITLE
Implement libGLES

### ISSUE THREAD

**Issue by kbrenneman** [NVIDIA/libglvnd#44]
Right now we've got placeholders for the libGLESv1.so and libGLESv2.so libraries, but nothing in them.

These should only need to be wrappers around libGLdispatch, just like libOpenGL.so is. However, they'll need to export a different set of symbols than libOpenGL.so, which will require some changes to the code generation.

**Comment by cubanismo** [comment]
Agreed.  The harder part, and what is more likely to block Mesa I think, is inclusion of a libEGL.so.1 dispatcher.

**Comment by kbrenneman** [comment]
True. I've filed issue #45 for defining and implementing an ABI for libEGL.

**Comment by kbrenneman** [comment]
Okay, I've checked in an implementation of both libGLESv1_CM.so.1 and libGLESv2.so.2. Both of them are just variations of libOpenGL.so with different export lists.

For libGLESv1_CM.so, I added in all of the GLES 1.0 and 1.1 functions. For libGLESv2.so, I added all of the core GLES functions from versions 2.0 through 3.2, inclusive.

I'm closing this issue. If it turns out we need to expand the export list for either of those libraries, then we can re-open it or open a new one.
